src/github.com/getlantern/eventual/eventual_test.go

```diff
@@ -14,6 +15,7 @@ const (
 )
 
 func TestSingle(t *testing.T) {
+	goroutines := runtime.NumGoroutine()
```

Comment:
Would increasing sleep time work? Just want a way to capture leaked goroutines. Another way would be to `pprof.Lookup("goroutine")` and compare entries one by one.